### PR TITLE
Add Apple Security

### PR DESCRIPTION
**Pull request by BlackSquirrelz**
**[awesome-apple-security](https://github.com/BlackSquirrelz/awesome-apple-security)**

**Apple Security is a relatively niche field in cyber security, getting started with it is requires a high amount of personal research. This list aims to make it easier for newcomers and more experienced infosec professionals to find valuable information, resources, tools and techniques related to Apple Security. I hope this list will help people to do security research and quickly gain a foothold in this topic.**

### By submitting this pull request I confirm I've read and complied with the below requirements 🖖

**Please read it multiple times. I spent a lot of time on these guidelines and most people miss a lot.**

## Requirements for your pull request

- [X] **Don't waste my time.** Do a good job, adhere to all the guidelines, and be responsive.

- [X]  **You have to review at least 2 other [open pull requests](https://github.com/sindresorhus/awesome/pulls?q=is%3Apr+is%3Aopen).**

	Try to prioritize unreviewed PRs, but you can also add more comments to reviewed PRs. Go through the below list when reviewing. This requirement is meant to help make the Awesome project self-sustaining. Comment here which PRs you reviewed. You're expected to put a good effort into this and to be thorough. Look at previous PR reviews for inspiration. **Just commenting “looks good” or simply marking the pull request as approved does not count!** You have to actually point out mistakes or improvement suggestions.

	Reviewed: #2257 #2263

- [X] You have read and understood the [instructions for creating a list](https://github.com/sindresorhus/awesome/blob/main/create-list.md).
- [X] This pull request has a title in the format `Add Name of List`.
	- ✅ `Add Swift`
	- ✅ `Add Software Architecture`
	- ❌ `Update readme.md`
	- ❌ `Add Awesome Swift`
	- ❌ `Add swift`
	- ❌ `add Swift`
	- ❌ `Adding Swift`
	- ❌ `Added Swift`
- [X] Your entry here should include a short description about the project/theme of the list. **It should not describe the list itself.** The first character should be uppercase and the description should end in a dot. It should be an objective description and not a tagline or marketing blurb.
	- ✅ `- [iOS](…) - Mobile operating system for Apple phones and tablets.`
	- ✅ `- [Framer](…) - Prototyping interactive UI designs.`
	- ❌ `- [iOS](…) - Resources and tools for iOS development.`
	- ❌ `- [Framer](…)`
	- ❌ `- [Framer](…) - prototyping interactive UI designs`
- [X] Your entry should be added at the bottom of the appropriate category.
- [X] The title of your entry should be title-cased and the URL to your list should end in `#readme`.
	- Example: `- [Software Architecture](https://github.com/simskij/awesome-software-architecture#readme) - The discipline of designing and building software.`
- [X] The suggested Awesome list complies with the below requirements.

## Requirements for your Awesome list

- [X] **Has been around for at least 30 days.**<br>That means 30 days from either the first real commit or when it was open-sourced. Whatever is most recent.
- [X] Don't open a Draft / WIP pull request while you work on the guidelines. A pull request should be 100% ready and should adhere to all the guidelines when you open it. **Instead use [#2242](https://github.com/sindresorhus/awesome/issues/2242) for incubation visibility**.
- [X] Run [`awesome-lint`](https://github.com/sindresorhus/awesome-lint) on your list and fix the reported issues. If there are false-positives or things that cannot/shouldn't be fixed, please [report it](https://github.com/sindresorhus/awesome-lint/issues/new).
- [X] The default branch should be named [`main`, not `master`](https://www.zdnet.com/article/github-to-replace-master-with-alternative-term-to-avoid-slavery-references/).
- [X] **Includes a succinct description of the project/theme at the top of the readme.** [(Example)](https://github.com/willempienaar/awesome-quantified-self)
	- ✅ `Mobile operating system for Apple phones and tablets.`
	- ✅ `Prototyping interactive UI designs.`
	- ❌ `Resources and tools for iOS development.`
	- ❌ `Awesome Framer packages and tools.`
- [X] It's the result of hard work and the best I could possibly produce.
	**If you have not put in considerable effort into your list, your pull request will be immediately closed.**
- [X] The repo name of your list should be in lowercase slug format: `awesome-name-of-list`.
	- ✅ `awesome-swift`
	- ✅ `awesome-web-typography`
	- ❌ `awesome-Swift`
	- ❌ `AwesomeWebTypography`
- [X] The heading title of your list should be in [title case](https://capitalizemytitle.com/) format: `# Awesome Name of List`.
	- ✅ `# Awesome Swift`
	- ✅ `# Awesome Web Typography`
	- ❌ `# awesome-swift`
	- ❌ `# AwesomeSwift`
- [X] Non-generated Markdown file in a GitHub repo.
-  [X] The repo should have `awesome-list` & `awesome` as [GitHub topics](https://help.github.com/articles/about-topics). I encourage you to add more relevant topics.
- [X] Not a duplicate. Please search for existing submissions.
- [X] Only has awesome items. Awesome lists are curations of the best, not everything.
- [X] Does not contain items that are unmaintained, has archived repo, deprecated, or missing docs. If you really need to include such items, they should be in a separate Markdown file.
- [X] _(Don't have a logo)_ Includes a project logo/illustration whenever possible.
	- Either centered, fullwidth, or placed at the top-right of the readme. [(Example)](https://github.com/sindresorhus/awesome-electron)
	- The image should link to the project website or any relevant website.
	- **The image should be high-DPI.** Set it to maximum half the width of the original image.
- [X] Entries have a description, unless the title is descriptive enough by itself. It rarely is though.
- [X] Includes the [Awesome badge](https://github.com/sindresorhus/awesome/blob/main/awesome.md#awesome-badge).
	- Should be placed on the right side of the readme heading.
		- Can be placed centered if the list has a centered graphics header.
	- Should link back to this list.
- [X] Has a Table of Contents section.
	- Should be named `Contents`, not `Table of Contents`.
	- Should be the first section in the list.
	- Should only have one level of [nested lists](https://commonmark.org/help/tutorial/10-nestedLists.html), preferably none.
	- Must not feature `Contributing` or `Footnotes` sections.
- [X] Has an appropriate license.
	- **We strongly recommend the [CC0 license](https://creativecommons.org/publicdomain/zero/1.0/), but any [Creative Commons license](https://creativecommons.org/choose/) will work.**
		- Tip: You can quickly add it to your repo by going to this URL: `https://github.com/<user>/<repo>/community/license/new?branch=main&template=cc0-1.0` (replace `<user>` and `<repo>` accordingly).
	- A code license like MIT, BSD, Apache, GPL, etc, is not acceptable. Neither are WTFPL and [Unlicense](https://unlicense.org).
	- Place a file named `license` or `LICENSE` in the repo root with the license text.
	- **Do not** add the license name, text, or a `Licence` section to the readme. GitHub already shows the license name and link to the full text at the top of the repo.
	- To verify that you've read all the guidelines, please comment on your pull request with just the word `unicorn`.
- [X] Has [contribution guidelines](https://github.com/sindresorhus/awesome/blob/main/awesome.md#include-contribution-guidelines).
	- The file should be named `contributing.md`. Casing is up to you.
	- It can optionally be linked from the readme in a dedicated section titled `Contributing`, positioned at the top or bottom of the main content.
	- The section should not appear in the Table of Contents.
- [X] All non-important but necessary content (like extra copyright notices, hyperlinks to sources, pointers to expansive content, etc) should be grouped in a `Footnotes` section at the bottom of the readme. The section should not be present in the Table of Contents.
- [X] Has consistent formatting and proper spelling/grammar.
	- The link and description are separated by a dash. <br>Example: `- [AVA](…) - JavaScript test runner.`
	- The description starts with an uppercase character and ends with a period.
	- Consistent and correct naming. For example, `Node.js`, not `NodeJS` or `node.js`.
- [X] Doesn't use [hard-wrapping](https://stackoverflow.com/questions/319925/difference-between-hard-wrap-and-soft-wrap).
- [X] Doesn't include a Travis badge.<br>You can still use Travis for list linting, but the badge has no value in the readme.
- [X] Doesn't include an `Inspired by awesome-foo` or `Inspired by the Awesome project` kinda link at the top of the readme. The Awesome badge is enough.

**Go to the top and read it again.**

### CODE REVIEW

**Comment by BlackSquirrelz** [comment]
unicorn

**Comment by Delft-Open-Hardware** [comment]
Hi BlackSquirrelz,
Thanks for the pull request!
To be honest I could not find that many suggestions to change.
Linting works, page has a clear layout, The only things I could find are the following:
- At communities there are only twitter accounts, then you could also call it Twitter accounts.
-  Videos only has 1 entry, but it seems you made your own playlist.
But the most important thing is the following:
The commits from the last 2 days made it an Awesome list, even though the repo has been online for more than a month, it was not a awesome list, so I don't know if people with suggestions would have summited them.
Other than this it is cool!

**Comment by BlackSquirrelz** [comment]
Hi @Delft-Open-Hardware 

Thank you for your detailed review.

- Fair point, I have updated it to Twitter for now, can still change it at a later stage if other socialmedia profiles are added.
- Yes, indeed, I curated a list of videos from youtube, would you prefer single videos or shall I rename the header to something like "playlists"?

I do see your point, officially it was just a list, I intended to make it an awesome list, but didn't feel comfortable calling it awesome just yet until the quality was high enough, and an official PR was made. Perhaps it can be included after another 30 days then ^^

File: readme.md [hunk 1]
```diff
@@ -742,6 +742,7 @@
 - [GDPR](https://github.com/bakke92/awesome-gdpr#readme) - Regulation on data protection and privacy for all individuals within EU.
 - [DevSecOps](https://github.com/TaptuIT/awesome-devsecops#readme) - Integration of security practices into [DevOps](https://en.wikipedia.org/wiki/DevOps).
 - [Executable Packing](https://github.com/dhondta/awesome-executable-packing#readme) - Packing and unpacking executable formats.
+- [Apple Security](https://github.com/BlackSquirrelz/awesome-apple-security#readme) - Apple macOS, iOS related cyber security tools, techniques and resources.
```
Comment (sindresorhus) [inline readme.md:745]: The description should not describe what the list contains. It should describe the topic.

Comment (BlackSquirrelz) [reply readme.md:745]: Changed to: _Apple Security resources._

**Comment by BlackSquirrelz** [comment]
Changed Readme according to linter

**Comment by claui** [comment]
@BlackSquirrelz Very useful list!

A couple of issues:

- The [Collector](https://github.com/BlackSquirrelz/awesome-apple-security/blob/708bd92d719c3ab5013c17ea44f47bbd08e162a3/README.md?plain=1#L29) link points to [an unmaintained repository](https://github.com/yelp/osxcollector). I suggest that you at least note that fact in the description. Alternatively, consider dropping the entry.

- The [Security Documentation](https://github.com/BlackSquirrelz/awesome-apple-security/blob/708bd92d719c3ab5013c17ea44f47bbd08e162a3/README.md?plain=1#L36) link points to [an error page](https://support.apple.com/en-gb/guide/security/welcome/we).

- The [Report Vulnerabilities](https://github.com/BlackSquirrelz/awesome-apple-security/blob/708bd92d719c3ab5013c17ea44f47bbd08e162a3/README.md?plain=1#L37) link points to [an error page](https://support.apple.com/en-gb/HT20122).

- The [Apple Platform Security](https://github.com/BlackSquirrelz/awesome-apple-security/blob/708bd92d719c3ab5013c17ea44f47bbd08e162a3/README.md?plain=1#L39) link just redirects to [the Apple Support home page](https://manuals.info.apple.com/MANUALS/1000/MA1902/en_GB/apple-platform-security-guide-b.pd).

- The [Process Tree](https://github.com/BlackSquirrelz/awesome-apple-security/blob/708bd92d719c3ab5013c17ea44f47bbd08e162a3/README.md?plain=1#L101) link doesn’t point to Process Tree. Instead, it simply points to a snapshot of the [mac_apt](https://github.com/ydkhatri/mac_apt/tree/729630c8bbe7a73cce3ca330305d3301a919cb07) repository. But you already have an entry for mac_apt.

File: readme.md [hunk 2]
```diff
@@ -751,6 +751,7 @@
 - [Executable Packing](https://github.com/dhondta/awesome-executable-packing#readme) - Packing and unpacking executable formats.
 - [Malware Persistence](https://github.com/Karneades/awesome-malware-persistence#readme) - Techniques that adversaries use to keep access to systems across restarts.
 - [EVM Security](https://github.com/kareniel/awesome-evm-security#readme) - Understanding the Ethereum Virtual Machine security ecosystem.
+- [Apple Security](https://github.com/BlackSquirrelz/awesome-apple-security#readme) - Apple Security resources.
```
Comment (sindresorhus) [inline readme.md:754]: The description is not valid.

**Comment by sindresorhus** [comment]
> - Twitter - macOS Security Researcher and Purple Teamer.

"Twitter" is moot here.

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**